akka-typed/src/main/scala/akka/typed/patterns/Receptionist.scala

```diff
@@ -72,18 +72,20 @@ object Receptionist {
 
   private type KV[K <: AbstractServiceKey] = ActorRef[K#Type]
 
-  private def behavior(map: TypedMultiMap[AbstractServiceKey, KV]): Behavior[Command] = Full {
-    case Msg(ctx, r: Register[t]) ⇒
+  private def behavior(map: TypedMultiMap[AbstractServiceKey, KV]): Behavior[Command] = Stateful[Command]({
+    case (ctx, r: Register[t]) ⇒
```

Comment:
NVM my brain inferred a `val` in front of the lambda, not sure what I was thinking.